.github/workflows/native_posix_run.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and run on native_posix
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container: zephyrprojectrtos/ci:latest
```

Comment:
isn't better to use specific version of this image? It may happen there is a new image version which require changes in zephyr repo which are not pulled into sdk-zephyr yet

Reply:
I really is better to use a specific version of the image. In fact, it is a recommended practice from docker as well. I was tempted to use the [v0.26.4](https://hub.docker.com/layers/zephyrprojectrtos/ci/v0.26.4/images/sha256-4b50d0c970fae7e2cd62a1b0a781d44436a203458c07d7d301631b6560882899?context=explore) tag but I was not entirely sure how zephyr's images are maintained. For example, will they get deleted in the future and so on. Hence I did not go for that approach. But definitely a good point.

Reply:
On second thought, I think it is more risky to depend on `latest` than to depend on a tag. So I will revise this workflow to use a tag instead.